src/ReactWrapper.jsx

```diff
@@ -885,12 +885,23 @@ class ReactWrapper {
   }
 
   /**
-   * Returns true if the current wrapper has no nodes. False otherwise.
+   * Delegates to exists()
    *
    * @returns {boolean}
    */
   isEmpty() {
-    return this.length === 0;
+    // eslint-disable-next-line no-console
+    console.warn('Deprecated method isEmpty() called, use exists() instead.');
```

Comment:
With a library like this, I don't see it making a difference. Aliasing an API without a log is not going to encourage anyone to remove it from existing tests and probably won't even affect them writing new tests. The odds of developers stumbling upon the documentation to see it's deprecated and actually act on that is unlikely.

If we add the log, it lets us remove the tech debt sooner. But I do think we should change this to only log once.